LICENSE.md

```diff
@@ -0,0 +1,6 @@
+## CC-BY 4.0 License
```

Comment:
After offline disussion with @ruimarinho and @fixe, we agreed that [CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/) may be more appropriate. Please make the changes in this file and in the README.